Makefile

```diff
@@ -40,6 +40,12 @@ acceptance-web: build
 test:
 	go test -race -cover ./...
 
+
+.PHONY: test-component
```

Comment:
I tried to run `make test-component` and I see a permission denied error:
```
go test -race -cover -component | grep -v created_at
# github.com/ONSdigital/dp-dataset-api
/Users/david/go/pkg/mod/github.com/hashicorp/go-immutable-radix@v1.2.0/iradix.go:7:2: open /Users/david/go/pkg/mod/github.com/hashicorp/golang-lru@v0.5.4/simplelru: permission denied
FAIL	github.com/ONSdigital/dp-dataset-api [setup failed]
```
- Do I need to authenticate somewhere in order to be able to use this dependency?
- Will this mean that we need to update the CI scripts as well?

Reply:
I've not seen that happen before :(

Reply:
it was a folder owner issue. Solved

Reply:
We need to document that your local mongoDB service shoud be stopped, if you have it running.